### PR TITLE
Add /weather page

### PR DESCRIPTION
**Pull request by lynnandtonic**
This PR adds a page that shows the weather for Phoenix, Tucson, and Flagstaff

image

### CODE REVIEW

**Comment by lynnandtonic** [comment]
I'm seeing some inaccuracies with the data. Will look into it.

**Comment by lynnandtonic** [comment]
Looks like it's showing highs/lows only. Will need to fetch different data to get current temps/conditions.

File: src/views/weather/index.pug [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        super();
+        this.response = {};
+      }
+
+      firstUpdated() {
+        fetch(
+          `https://api.weather.gov/points/${this.latitude},${this.longitude}`
+        )
+          .then(r => r.json())
+          .then(r => r.properties.forecast)
```
Comment (rtorr) [inline src/views/weather/index.pug]: try `properties.forecastHourly` instead

**Comment by lynnandtonic** [comment]
Thank you, @rtorr!

**Pull request merged**